challenge/src/components/JojoStand.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+
+class JojoStand extends React.Component{
+    constructor(props){
+        super(props);
+    }
+    visibility = () => {
+        if(!this.props.isVisible){
+            return "scale-down-center"
+        }else{
+            return ""
+        }
+    }
+    render(){
+        const standStyling = {
+            margin: "5%",
+            marginTop: "1%",
+            marginBottom: "0",
+            width: "25%"
+        }
+        var stands = {
```

Comment:
perhaps we can make this object a prop for the about component, or just move it to the about component's redner method. thin take in a `stand` and `url` prop for this component